src/main.js

```diff
@@ -10,6 +10,7 @@ function onSubmit(event) {
 
   const formValue = [...form.elements]
     .filter(({ type }) => !INPUT_BUTTON_TYPES.includes(type))
+    // eslint-disable-next-line
     .reduce((acc, { value, name }) => ({ ...acc, [name]: value }), {});
```

Comment:
```suggestion
    .reduce((acc, { value, name }) => ({
      ...acc, [name]: value
    }), {});
```
зроби так і тоді видали коментар